### DISCUSSION TITLE
How to add HTML draggable="false" to extensions? HTMLAttributes doesn't work.

### DISCUSSION THREAD

**Discussion by gustavotoyota** [ueberdosis/tiptap#2972] [General Questions]
I tried adding `HTMLAttributes: { draggable: false }` to the configuration but it doesn't set draggable="false".
HTMLAttributes allows me to add classes, but not change HTML attributes.
I also tried extending the extension like `HorizontalRule.entend({ draggable: false })`, but that also doesn't set draggable="false".

Thank you.

**Comment by sereneinserenade** [comment]
@gustavotoyota HTMLAttributes is an object of string keys and string values(in TS terms `Record<string, string>`), booleans might not work. 

You can try this out
```ts
HorizontalRule.entend({ draggable: "false" })
```
